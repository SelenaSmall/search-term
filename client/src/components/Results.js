import React from 'react'
import {Link } from 'react-router-dom'
import Header from './Header';
import Footer from './Footer';

const Results = ({location: { state: { score }} }) => (
  <div>
    <Header />

    <section id="middle-region">
      <div className="game-main-container text-center">
        <div>
        <h1 className="congratulations">Congratulations!</h1>
        <h2 className="final-score">You scored {score}</h2>

        <Link to="/" className="game-button">Home</Link>
        </div>
      </div>
    </section>

    <Footer />
  </div>
)

export default Results