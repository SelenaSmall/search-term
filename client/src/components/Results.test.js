import React from 'react';
import Results from './Results';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

it('basic rendering', () => {
  const output = shallow(<Results location={{state: {score: null}}} />)
  expect(shallowToJson(output)).toMatchSnapshot()
})
