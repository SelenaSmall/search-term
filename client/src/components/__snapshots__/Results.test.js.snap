// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic rendering 1`] = `
<div>
  <Component />
  <section
    id="middle-region"
  >
    <div
      className="game-main-container text-center"
    >
      <div>
        <h1
          className="congratulations"
        >
          Congratulations!
        </h1>
        <h2
          className="final-score"
        >
          You scored 
        </h2>
        <Link
          className="game-button"
          replace={false}
          to="/"
        >
          Home
        </Link>
      </div>
    </div>
  </section>
  <Component />
</div>
`;
